src/apps/hello-world/hello-world.entry.tsx

```diff
@@ -8,8 +8,6 @@ export interface HelloWorldEntryProps {
   whatText: string;
 }
 
-const HelloWorldEntry: React.FC<HelloWorldEntryProps> = (
-  props: HelloWorldEntryProps
-) => <HelloWorld />;
+const HelloWorldEntry: React.FC<HelloWorldEntryProps> = () => <HelloWorld />;
```

Comment:
Why do `HelloWord` change by adding an API client?

Reply:
I think it is leftovers from #30 where the generated client was used in the Hello World app.